imagine-dragons-web/home.html

```diff
@@ -5,17 +5,17 @@
   <head>
     <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
     <title>The website of Imagine Dragons</title>
+    <link rel="stylesheet" type="text/css" href="resources\css\stylesheet.css">
+    <link rel="stylesheet" type="text/css" href="resources\css\home-stylesheet.css">
   </head>
 
   <body>
     <div id="nav-placeholder"></div>
```

Comment:
Add a common title to all your pages "Imagine Dragons"

Reply:
I am not sure what is the differnce between this comment and your previous comment "Each page should display the page title. - change this in all pages"